addons/sale/static/src/views/sale_onboarding_kanban/sale_onboarding_kanban_renderer.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates xml:space="preserve">
+    <t t-name="sale.SaleKanbanRenderer" t-inherit="web.KanbanRenderer" t-inherit-mode="primary">
+        <t t-call="web.ActionHelper" position="inside">
+            <SaleActionHelper noContentHelp="props.noContentHelp"/>
+        </t>
+        <t t-call="web.ActionHelper" position="attributes">
+            <attribute name="t-call"></attribute>
+        </t>
```

Comment:
Is that really what you want to do? With this code, you'll show the `SaleActionHelper` only if `props.noContentHelp` is defined. My guess is that you always want to show `SaleActionHelper` no matter the `props.noContentHelp`.
```suggestion
        <t t-call="web.ActionHelper" position="replace">
            <SaleActionHelper noContentHelp="props.noContentHelp"/>
        </t>
```

Reply:
`t-call="web.ActionHelper"` is actually `t-if` and there is `t-else` related to this `t-if` so either we have to keep what I did before [Here](https://github.com/odoo/odoo/pull/152270#discussion_r1591172627) or we have to do this if we really don't want to add `t-if="showNoContentHelper"`

Yes we do want to display it if no props is defined but I need to add some `t-if` here 
So either add `t-if="showNoContentHelper"` or `t-if=true` :thought_balloon:

Reply:
Then I'd rather replace directly `<t t-if="showNoContentHelper">` than adding a `t-if=true`
```suggestion
         <t t-if="showNoContentHelper" position="replace">
            <t t-if="showNoContentHelper">
                <SaleActionHelper noContentHelp="props.noContentHelp"/>
            </t>
        </t>
```